src/screens/Login/Login.js

```diff
@@ -2,7 +2,9 @@ import { useTheme } from '@react-navigation/native';
 import React, { useState } from 'react';
 import { View } from 'react-native';
 import { shallowEqual, useDispatch, useSelector } from 'react-redux';
+import { generateSessionId} from '@/actions/UserActions';
 import { login, TYPES } from '@/actions/UserActions';
+//import { se, TYPES } from '@/actions/UserActions';
```

Comment:
Remove commented code. Here and everywhere (when possible).

Reply:
ok!